languages/es.json

```diff
@@ -0,0 +1,208 @@
+    "Waiting Confirmation": "Esperando confirmación",
+    "Subdomain added": "Subdominio agregado",
+    "Subdomain Name": "Nombre del subdominio",
+    "Owner Address (Optional)": "Dirección del Propietario (Opcional)",
+    "Type address": "Dirección del  tipo",
+    "The Resolver is a Smart Contract responsible for the process of translating names into addresses. You can select a public resolver or a custom resolver.": "El Resolver es el contrato inteligente responsable del proceso de traducir nombres en direcciones. Puede seleccionar un resolver público o uno personalizado",
+    "active": "activo",
+    "expiring": "por vencerse",
+    "expired": "vencido",
+    "Pending...": "Pendiente...",
+    "Register Now!": "Registrar Ahora!",
+    "Domain detail page still in progress for this status": "La página de detalle del dominio continua en progreso en el siguiente estado",
+    "Your domain is expired": "Tu dominio esta vencido",
+    "Your domain is expiring": "Tu dominio esta por vencerse",
+    "Are you sure you want to close channel {{channelIdentifier}} with partner {{partner}} for token {{tokenName}}?": "¿Está seguro que desea cerrar el canal {{channelIdentifier}} con el partner {{partner}} para el  token {{tokenName}}?",
+    "Requesting Close Channel": "Solicitando cierre del canal",
+    "Channel Closed Successfully!": "Canal cerrado satisfactoriamente",
+    "Unknown Error trying to close channel": "Error desconocido intentado cerrar el canal",
+    "{{tokenSymbol}} Amount": "Cantidad de {{tokenSymbol}}",
+    "Deposit Requested": "Deposito solicitado",
+    "Deposit Done Successfully": "Deposito realizado satisfactoriamente",
+    "Unknown Error trying to deposit": "Error desconocido intentando depositar",
+    "Updating chain address": "Actualizando dirección de la cadena",
+    "Address cannot be empty": "La dirección no puede ser vacía",
+    "Deleting chain address": "Eliminando dirección de la cadena",
+    "Addresses": "Dirección",
+    "No addresses found": "No se encontraron direcciones",
+    "Add Address": "Agregar dirección",
+    "pending": "pendiente",
+    "Unknown Error trying to close channel!": "Error desconocido cerrando el canal",
+    "Close": "Cerrar",
+    "Please Wait...": "Por favor, espere",
+    "Unknown Error trying to deposit!": "Error desconocido intentando depositar",
+    "Deposit on Channel": "Depositar en canal",
+    "Are you sure you want to deposit {{amount}} {{tokenSymbol}} tokens on channel {{channelIdentifier}} with partner {{destination}}?": "¿Está seguro que quiere depositar {{amount}} {{tokenSymbol}} en el canal {{channelIdentifier}} con el partner {{destination}}?",
+    "Destination has to be a valid domain name or checksum address.": "El destino debe ser un dominio válido o una checksum address",
+    "Making deposit\nPlease wait, this operation could take around 8 minutes.": "Depositando\nPor favor, espere\n esta operación puede tardar alrededor de 8 minutos.",
```

Comment:
Second \n i think we can delete it

Reply:
The width of the line is greater than viewport's width. I think the best approach would be to delete the \n as yo say, and also remove " white-space: pre;". from css class .app-loader-message, in order to break the line properly.